sdcm/cluster_aws.py

```diff
@@ -479,13 +479,21 @@ def get_console_output(self):
         output = self._ec2_service.meta.client.get_console_output(
             InstanceId=self._instance.id,
         )
-        return output['Output']
```

Comment:
It might be cleaner to use ‘output.get('Output', '')‘ in those case. 
but that's more of a taste/style

Reply:
(doing formating from my phone isn't a smart idea)

Reply:
Fix